prov/tcp/src/tcpx_ep.c

```diff
@@ -185,6 +200,42 @@ static int tcpx_ep_shutdown(struct fid_ep *ep, uint64_t flags)
 	return ret;
 }
 
+static int tcpx_pep_bind_to_port_range(struct tcpx_pep *pep)
```

Comment:
This function can be generalized.  Instead of passing in the passive EP, pass in the socket and src address.  This will make it usable with active EPs, and make it easier to extract into common code if other providers need the same functionality.